### PR TITLE
PIM-11067: fix display banner when user does not have system configur…

### PR DESCRIPTION
**Pull request by marinfazioakeneo**
…ation acl

**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: front-packages/shared/src/dependencies/system-configuration.ts [hunk 1]
```diff
@@ -17,8 +17,6 @@ const systemConfiguration: SystemConfiguration = {
 
       return Promise.resolve();
     } catch (error) {
-      systemConfigurationData['sandbox_banner'] = true;
```
Comment (StevenVAIDIE) [inline front-packages/shared/src/dependencies/system-configuration.ts:20]: Can you revert this change ? In this way we will display the banner by default if there is another problem

**Review by StevenVAIDIE — APPROVED** [review]
Do not forget to change the CHANGELOG.md by adding a new line

File: front-packages/shared/src/dependencies/system-configuration.ts (hunk 1 above)
Comment (marinfazioakeneo) [reply front-packages/shared/src/dependencies/system-configuration.ts]: done

**Review by ValentinMumble — APPROVED** [review]
